packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -397,14 +397,17 @@ const ProductList = (props) => {
         navigate(newPath)
     }
 
-    // Create reusable StoreInventoryFilter component to avoid repetition
-    const storeInventoryFilterComponent = (
```

Comment:
Updated this to an arrow function because I believe this was causing the error we were seeing on the PLP![image](https://github.com/user-attachments/assets/09f05ad3-2441-4c48-af86-0d9530b947e1)